src/AdminConsole/Billing/Configuration/StripeOptions.cs

```diff
@@ -6,5 +6,7 @@ public class StripeOptions
 
     public string ApiKey { get; set; }
 
+    public ICollection<string> OnSale { get; set; }
```

Comment:
nitpick: Perhaps use a different term, this signals to me that it's "on sale" as in 50% off.